### ISSUE TITLE
Localization Loss

### ISSUE THREAD

**Issue by cyberphantom** [pierluigiferrari/ssd_keras#159]
Hi @pierluigiferrari 
I really liked your implementation and that is why I want to follow it and analyze the original SSD paper

In the Training objective section in the paper there is a part that I couldn't find in the keras_ssd_loss.py file which is related to the regressing the offsets for the center (cx, cy) of the default box (d) and for its width (w) and height (h). In localization loss we are using the L1 smooth between default box parameters (l) and ground truth parameters (g)
 
![image](https://user-images.githubusercontent.com/8687562/43038953-214ca8fe-8cf1-11e8-8646-7597be4fdf62.png)

 Can you please comment on that?

Thanks,

**Comment by pierluigiferrari** [comment]
The offsets between ground truth and default box coordinates are computed in the [`SSDInputEncoder`](https://github.com/pierluigiferrari/ssd_keras/blob/3ac9adaf3889f1020d74b0eeefea281d5e82f353/ssd_encoder_decoder/ssd_input_encoder.py#L393).

**Comment by cyberphantom** [comment]
Thank you, I always wondered what is this encoder for :)
